vermouth/dssp/dssp.py

```diff
@@ -194,10 +195,14 @@ def run_dssp(system, executable='dssp', savefile=None, defer_writing=True, versi
         Parse a DSSP output.
     """
     # check version
-    process = subprocess.run(["dssp", "--version"], stdout=subprocess.PIPE, stderr=subprocess.PIPE)
-    version_found = process.stdout.decode('UTF8')
-    if version not in version_found:
-        raise DSSPError('Vermouth currently only supports DSSP version 3.0.0.')
+    supported_versions = [2, 3]
```

Comment:
I think this should be a global immutable constant, e.g. `SUPPORTED_DSSP_VERSIONS = (2, 3)` at the top of the file.